src/main/scala/com/mwz/sonar/scala/scoverage/ScoverageMetrics.scala

```diff
@@ -0,0 +1,94 @@
+package com.mwz.sonar.scala.scoverage
```

Comment:
I think we need to retain the original copyright in each of those files unless you completely rewrote those from scratch.

Reply:
I'm almost 100% sure that the files with the new copyright had been rewritten almost from scratch.
But if you have in mind any particular file that should retain the old copyright I will change it immediately.

Reply:
Yeah cool, in that case could you then add our standard copyright instead?
```
/*
 * Sonar Scala Plugin
 * Copyright (C) 2018 All contributors
...
```

Reply:
Yeah I had already done that in a new commit, I missed it.